<launch>
  <arg name="target_frame" default="tracker_1" />
	<node pkg="iri_htc_vive_tracker"
		type = "iri_htc_vive_tracker"
		name = "iri_htc_vive_tracker" 
        output = "screen">
         <param name="~target_frame_name" type="string" value="$(arg target_frame)"/>
	</node>
</launch>
